src/inspect_ai/model/_providers/anthropic.py

```diff
@@ -805,11 +774,13 @@ async def model_output_from_message(
             content.append(ContentText(type="text", text=content_text))
         elif isinstance(content_block, ToolUseBlock):
             tool_calls = tool_calls or []
+            info = maybe_mapped_call_info(content_block.name, tools)
             tool_calls.append(
                 ToolCall(
-                    type="function",
+                    type=info.native_type,
                     id=content_block.id,
-                    function=maybe_map_from_native_tool(content_block.name, tools),
+                    function=info.inspect_name,
+                    native_name=info.native_name,
```

Comment:
This is the _forward_ mapping. From the native/model name to the inspect name and native/model type.